www/main/settings/components/Settings.jsx

```diff
@@ -5,15 +5,22 @@ import { Overlay } from '../../common/sttm-ui';
 
 import SettingsNav from './SettingsNav';
 import SettingsContainer from './SettingsContainer';
-
+import ThemeContainer from './ThemeContainer';
+import SettingViewer from './SettingViewer';
 import { settingsNavObj, settingsObj } from '../utils';
 
 const Settings = ({ onScreenClose }) => {
   return (
     <Overlay onScreenClose={onScreenClose}>
```

Comment:
I don't like how the settings aren't fluid. They're a static size that get scroll bars. We can do better than that on smaller screen.

Reply:
@maneetpaul @prebhdevsingh  
possible to get designs for this.. I can send some options as well.. 🙏